diff --git a/index.js b/index.js
index 501f80d347a4397d473dcee1d0567b6e1f694ec7..6a3f662a4c286d456a87735eaf070fc7f97790e8 100644
--- a/index.js
+++ b/index.js
@@ -157,7 +157,7 @@ function customDecodeURIComponent(input) {
 	return input;
 }
 
-export default function decodeUriComponent(encodedURI) {
+module.exports = function decodeUriComponent(encodedURI) {
 	if (typeof encodedURI !== 'string') {
 		throw new TypeError(`Expected \`encodedURI\` to be of type \`string\`, got \`${typeof encodedURI}\``);
 	}
@@ -169,4 +169,4 @@ export default function decodeUriComponent(encodedURI) {
 		// Fallback to a more advanced decoder
 		return customDecodeURIComponent(encodedURI);
 	}
-}
+};
diff --git a/package.json b/package.json
index 8d01f060a3a2da1ee19861badf97d0b5db7192f6..3a255963afeac34f0246dbd9b75bc098cbae70ad 100644
--- a/package.json
+++ b/package.json
@@ -10,11 +10,8 @@
 		"email": "sam.verschueren@gmail.com",
 		"url": "github.com/SamVerschueren"
 	},
-	"type": "module",
-	"exports": {
-		"types": "./index.d.ts",
-		"default": "./index.js"
-	},
+	"main": "./index.js",
+	"types": "./index.d.ts",
 	"engines": {
 		"node": ">=14.16"
 	},
